lib/oli_web/live/workspaces/instructor/index_live.ex

```diff
@@ -433,4 +426,23 @@ defmodule OliWeb.Workspaces.Instructor.IndexLive do
         Params.get_boolean_param(params, "sidebar_expanded", @default_params.sidebar_expanded)
     }
   end
+
+  defp sections_where_user_is_instructor(user_id) do
```

Comment:
Nit, but what about moving this and other queries to a separate context module?

Reply:
Agree, we should not really be calling Oli.Repo from OliWeb context or making queries. This would be better suited in the Oli.Delivery.Sections module

Reply:
Agree with Gaston. When you move it you can also add some tests to verify the expected behavior.

Reply:
👍 Moved them to the `Oli.Delivery.Sections` module